runtime/src/lib.rs

```diff
@@ -608,12 +608,13 @@ where
     }
 }
 
-impl DispatchAggregation for Runtime {
+impl DispatchAggregation<Balance> for Runtime {
     fn dispatch_aggregation(
         domain_id: u32,
         aggregation_id: u64,
         aggregation: H256,
         destination_params: Destination,
+        fee: Balance,
     ) -> DispatchResult {
         match destination_params {
             Destination::None => Ok(()),
```

Comment:
Shouldn't we take the fee from the account of the delivery owner ? Here is still set to the Account ID of the hyperbridge_aggregations pallet

Reply:
right! will fix this.